draft-ietf-quic-transport.md

```diff
@@ -2070,6 +2150,11 @@ Connection ID:
 
 : A 64-bit connection ID.
 
+Stateless Reset Token:
+
+: A 128-bit value that will be used to for a stateless reset when the associated
+  connection ID is used (see {{stateless-reset}}).
```

Comment:
yes, it needs to be a new token otherwise it is not possible to hash 5-tuple based on an endpoint key and then multiple connections would have to share token.

Is there any text that says the token MUST be unique to the path with high probality.